templates/magazine/moderators.html.twig

```diff
@@ -14,22 +14,42 @@
 
 {% block body %}
     <h1>{{ 'moderators'|trans }}</h1>
-    {% if not app.user or not magazine.userIsModerator(app.user) %}
-        <form action="{{ path('magazine_moderator_request', {name: magazine.name}) }}"
-              method="POST"
-              class="float-end mb-2"
-              onsubmit="return confirm('{{ 'are_you_sure'|trans }}');">
-            <input type="hidden" name="token" value="{{ csrf_token('moderator_request') }}">
-            <button type="submit" class="btn btn__secondary">
-                <i class="fa-solid fa-hand-point-up"></i>
-                {% if not app.user or not app.user.hasModeratorRequest(magazine) %}
-                    <span>{{ 'apply_for_moderator'|trans }}</span>
-                {% else %}
-                    <span>{{ 'cancel_request'|trans }}</span>
-                {% endif %}
-            </button>
-        </form>
-    {% endif %}
+    <div class="flex">
+        {% if not app.user or not magazine.userIsModerator(app.user) %}
+            <form action="{{ path('magazine_moderator_request', {name: magazine.name}) }}"
+                  method="POST"
+                  class="float-end mb-2"
+                  onsubmit="return confirm('{{ 'are_you_sure'|trans }}');">
+                <input type="hidden" name="token" value="{{ csrf_token('moderator_request') }}">
+                <button type="submit" class="btn btn__secondary">
+                    <i class="fa-solid fa-hand-point-up"></i>
+                    {% if not app.user or not app.user.hasModeratorRequest(magazine) %}
+                        <span>{{ 'apply_for_moderator'|trans }}</span>
+                    {% else %}
+                        <span>{{ 'cancel_request'|trans }}</span>
+                    {% endif %}
+                </button>
+            </form>
+        {% endif %}
+        {% if not app.user or not magazine.userIsOwner(app.user) and magazine.isAbandoned() %}
```

Comment:
I guess we'll likely want the same change on this line maybe

```
{% if not app.user or (not magazine.userIsOwner(app.user) and app.user.visibility is same as 'visible') %}
```

tbh I didn't even notice this page existed, that you could apply to be a moderator at any time, didn't realize the finger wag icon was a link

Reply:
What do you exactly mean with "non-visible accounts"?

Reply:
Check out #253 which has the changes I described going into this branch. It just means, as far as I know, accounts that request to be deleted and accounts that are suspended can apply for moderator or request ownership of magazines (or rather, as you said, see the buttons, they can still get around it)